.github/paths-labeller.yml

```diff
@@ -0,0 +1,53 @@
+---
+- "beats":
```

Comment:
Should we just have a `schemas` label to cover beats + ecs?

Reply:
ok - I will remove those labels and update